src/components/DisplayEmbedTag/DisplayExternal.jsx

```diff
@@ -171,24 +154,19 @@ export class DisplayExternal extends Component {
     }
 
     return (
-      <Fragment>
-        <div>
-          <CrossButton
-            css={colorFigureButtonsLinkStyle(colors.support.red)}
-            stripped
-            onClick={onRemoveClick}
-          />
-          {allowedProvider.name && (
-            <Button
-              css={colorFigureButtonsLinkStyle(colors.support.green)}
-              stripped
-              onClick={() =>
-                this.openEditEmbed(allowedProvider.name.toLowerCase())
-              }>
-              <Pencil />
-            </Button>
-          )}
-        </div>
+      <div className="c-figure">
+        <FigureButtons
+          tooltip={t('form.video.remove')}
+          onRemoveClick={onRemoveClick}
+          embed={embed}
+          t={t}
```

Comment:
Vits å sende ned t? Og ikke bruker ndla-i18n?

Reply:
Fordi jest feilet for nestede komponenter som tok inn ndla-i18n